lazyflow/roi.py

```diff
@@ -751,7 +751,7 @@ def determine_optimal_request_blockshape(
     ideal_blockshape = numpy.asarray(ideal_blockshape)
 
     target_block_volume_bytes = available_ram // num_threads
-    target_block_volume_pixels = target_block_volume_bytes // ram_usage_per_requested_pixel
+    target_block_volume_pixels = max(4096, target_block_volume_bytes // ram_usage_per_requested_pixel)
```

Comment:
Is this the code that could output 0 and cause that "division by 0" error? Seems like this would happen only if `available_ram` was set to 0. But in that case, shouldn't this function throw an exception? Or just assume that it was a caller error?

Reply:
No, it's integer division so for example ```100 // 300 == 0```

Reply:
Ah, of course!